### PR TITLE
Added meta-olp-edge-sdk layer for Yocto integration

### PR DESCRIPTION
**Pull request by kmanikandan-kmk**

### CODE REVIEW

File: docs/dataservice-write-example.md [hunk 1]
```diff
@@ -109,6 +109,29 @@ Select the `dataservice-write-example` scheme from the schemes list in the `XCod
 
 Once everything is correctly set up, build and run the example application on your device and you should see the `Publish Successful` message in the main UI screen. If you encountered an error message, e.g. `Publish Failed`, please check the device's logs for the detailed description of the error.
```
Comment (ksenteer) [inline docs/dataservice-write-example.md]: I suggest to rephrase as follows:

Once everything is set up correctly, build and run the example application on your device. After building and running of the example is complete, you see the `Publish Successful` message in the main UI screen. If you encounter an error message, e.g. `Publish Failed`, please check the device's logs for the detailed description of the error.

File: external/leveldb/CMakeLists-leveldb.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
```
Comment (ksenteer) [inline external/leveldb/CMakeLists-leveldb.txt]: governed by "the" BSD-style license

File: docs/dataservice-write-example.md (hunk 1 above)
Comment (kmanikandan-kmk) [reply docs/dataservice-write-example.md]: These are not my changes and have been there even before my change. Could you please give comments on my changes?

File: external/leveldb/CMakeLists-leveldb.txt (hunk 2 above)
Comment (kmanikandan-kmk) [reply external/leveldb/CMakeLists-leveldb.txt]: The same as previous comment

File: external/leveldb/CMakeLists-leveldb.txt [hunk 3]
```diff
@@ -2,7 +2,7 @@
...
 # found in the LICENSE file. See the AUTHORS file for names of contributors.
 
-cmake_minimum_required(VERSION 3.9)
+cmake_minimum_required(VERSION 3.8)
```
Comment (serhiikobalt) [inline external/leveldb/CMakeLists-leveldb.txt]: LevelDB requires 3.9 I wonder it it will compile at all

File: docs/dataservice-write-example.md [hunk 4]
```diff
@@ -109,6 +109,29 @@ Select the `dataservice-write-example` scheme from the schemes list in the `XCod
...
 
+## Building and running on Embedded Device using Yocto project
+
+This example shows how to integrate and use the HERE OLP Edge SDK C++ in an Embedded device firmware built using Yocto project.
```
Comment (ksenteer) [inline docs/dataservice-write-example.md]: the Yocto project

File: docs/dataservice-write-example.md (hunk 1 above)
Comment (ksenteer) [reply docs/dataservice-write-example.md]: Looks good, no more comments from my side.

Comment (ksenteer) [reply docs/dataservice-write-example.md]: Looks good, no more comments from my side.

**Comment by r0busta** [comment]
Closing the PR as it was not touched by anyone for more than 30 days. Feel free to reopen.

**Pull request closed without merging**